Automattic/jetpack
# SEO: styles for suggested/max char count for front page meta description in wp-admin

#### Changes proposed in this Pull Request:

- When users type over the suggested char length, a warning background
color is shown.
- When users type up to the max char length, a different warning
background is shown in addition to outputting that max chars has been
reached.

#### Jetpack product discussion

* Internal: p8oabR-DS-p2#comment-5149

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* After pulling down these changes, in WP Admin navigate to `/wp-admin/admin.php?page=jetpack#/traffic`
* If SEO Tools are enabled, in the `Expand to edit your front page meta description.` section, you can type in a custom meta description. After typing 160 or more characters, a yellow background color should appear for the active character count. After typing 300 characters, the background should turn red, and text is show saying the max character count has been reached.

#### GIF

![Apr-08-2021 10-58-40](https://user-images.githubusercontent.com/15803018/114066946-7c4f1180-9859-11eb-9dd9-1ca5d69d10bb.gif)
